test/transformation-test.coffee

```diff
@@ -160,6 +166,15 @@ describe('Transformations', ->
             if type isnt 'refract'
               assert.equal(ast.sections[0].resources[0].responses[0].body, 'Hello World')
           )
+          it('response has headers1A property', ->
+            assert.isDefined(ast.sections[0].resources[0].responses[0].headers1A)
+          )
+          it('response has headers1A with three Set-Cookie headers', ->
+            setCookieHeaders = ast.sections[0].resources[0].responses[0].headers1A.filter((item) ->
+              item.name is 'Set-Cookie'
+            )
+            assert.equal(setCookieHeaders.length, 3)
```

Comment:
Could you please check that the value is different for all these? (Make sure it's not being overwritten)